configure.ac

```diff
@@ -4133,10 +4133,23 @@ case "$host" in
 esac
 
 case $with_libgdiplus in
-    no|installed)
+    no)
     libgdiplus_loc=
     ;;
 
+    installed)
+    libgdiplus_loc=
+    if test x$cross_compiling = xno; then
```

Comment:
1. It might make sense to be sysroot-sensitive, and therefore work with cross builds.
2. I'm not sure about the iOS/Android stories -- the problem I'm solving is hundreds/thousands of errors in make check by default.

Reply:
Re: 2.) libgdiplus isn't present on iOS/Android so that's fine